src/web/app/pages-help/instructor-help-page/instructor-help-page.component.ts

```diff
@@ -78,8 +78,10 @@ export class InstructorHelpPageComponent implements OnInit, AfterViewInit {
    * Filters the help contents and displays only those that matches the filter.
    */
   search(): void {
-    this.matchFound = 0;
     if (this.searchTerm !== '') {
```

Comment:
Let's try to reduce the indentation from the if statements:
```
    if (this.searchTerm === '') {
      this.clear();
      return;
    }

    const nxtKey = this.searchTerm.toLowerCase();
    if (nxtKey !== this.key) {
      this.matchFound = 0;
    }

    this.key = nxtKey;
```

Reply:
@rrtheonlyone have made the requested changes. Please review. Thanks!